apps/chronicles/screens/components/components/ButtonScreen.tsx

```diff
@@ -102,6 +102,8 @@ const themeStyles = EDSStyleSheet.create(theme => ({
     buttonRow: {
         paddingVertical: 8,
         flexDirection: "row",
+        flexWrap: "wrap",
         justifyContent: "space-around",
+        gap: 12,
```

Comment:
Is there a token spacing value we can use instead of this?
I see the padding vertical is also a hard-coded number, but I won't bother you with previously written code

Reply:
`gap` and `paddingVertical` now uses responsive values from `theme` in [d069be9](https://github.com/equinor/mad/pull/712/commits/d069be9eaf7473f379c8d6863f189a801bba55ba).